Extensions/ColorConversion.json

```diff
@@ -58,6 +57,58 @@
       ],
       "objectGroups": []
     },
+    {
+      "description": "Apply a weighted mean between 2 RGB colors.",
+      "fullName": "RGB mean",
+      "functionType": "StringExpression",
+      "name": "RgbMean",
+      "private": false,
+      "sentence": "",
+      "events": [
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": "try {\n    // split do allocations, but the code a lot simpler this way.\n    const components1 = eventsFunctionContext.getArgument(\"Color1\").split(\";\");\n    const r1 = gdjs.evtTools.common.clamp(parseFloat(components1[0]), 0, 255);\n    const g1 = gdjs.evtTools.common.clamp(parseFloat(components1[1]), 0, 255);\n    const b1 = gdjs.evtTools.common.clamp(parseFloat(components1[2]), 0, 255);\n\n    const components2 = eventsFunctionContext.getArgument(\"Color2\").split(\";\");\n    const r2 = gdjs.evtTools.common.clamp(parseFloat(components2[0]), 0, 255);\n    const g2 = gdjs.evtTools.common.clamp(parseFloat(components2[1]), 0, 255);\n    const b2 = gdjs.evtTools.common.clamp(parseFloat(components2[2]), 0, 255);\n\n    const ratio = eventsFunctionContext.getArgument(\"Ratio\");\n    const r = gdjs.evtTools.common.clamp(Math.round((1 - ratio) * r1 + ratio * r2), 0, 255);\n    const g = gdjs.evtTools.common.clamp(Math.round((1 - ratio) * g1 + ratio * g2), 0, 255);\n    const b = gdjs.evtTools.common.clamp(Math.round((1 - ratio) * b1 + ratio * b2), 0, 255);\n\n    eventsFunctionContext.returnValue = r + \";\" + g + \";\" + b;\n} catch (e) {\n    console.error(\"Unable to convert a color: \", e);\n    eventsFunctionContext.returnValue = \"0;0;0\";\n}",
+          "parameterObjects": "",
+          "useStrict": true,
+          "eventsSheetExpanded": true
+        }
```

Comment:
Is it truly necessary to use JavaScript here? Unless there is a very big impact on convenience (usage of a complex premade JavaScript code to not rewrite it), performance, or feasibility (for example when requiring APIs not exposed to GDevelop events), JavaScript should be avoided, since it makes the extension harder to review, modify and understand for all non-JavaScript-using users, which constitute the majority of the community. 
It is also written in the best practices: 
![image](https://user-images.githubusercontent.com/19349038/144755216-61170f86-ad22-4d6b-8f22-877c9c6e4276.png)

Reply:
Because this is a "tool" extension, might be worth keeping JS here.

Reply:
The rest of the extension is in JS too, it was easier to copy-paste code.